galeracluster/source/sslsst.rst

```diff
@@ -0,0 +1,150 @@
+
+The particular method you use to secure State Snapshot Transfer scripts depends upon the particular method you choose to use on your cluster.
+
+.. note:: For Gelera Cluster, SSL configurations are not dynamic.  Since they must be set on every node in the cluster, if you are enabling this feature with a running cluster you need to restart the entire cluster.
+
+
+----------------------------------
+Enabling SSL for ``mysqldump``
+----------------------------------
+.. _`ssl-mysqldump`:
+
+The procedure for securing ``mysqldump`` is fairly similar to that of securing the database server and client and the standard method of enabling :term:`State Snapshot Transfer` script on your node.  Given that it connects through the database client, you can use the same SSL certificates you created for replication traffic.  In order to enable this for the entire cluster, you need to repeat this process on every node in the cluster.  
+
+#. From the database client you need to create a user for the State Snapshot Transfer script.
+
+   .. code-block:: mysql
+
+      CREATE USER 'sst_user'$'%' IDENTIFIED BY PASSWORD 'sst_password';
+
+   Bear in mind that, due to the manner in which the SST script is called, the user name and password must be the same on all nodes.
+
+#. Grant privileges to this user and require SSL.
+
+   .. code-block:: mysql
+
+      GRANT ALL ON *.* TO 'sst_user'@'%' REQUIRE SSL;
+
+#. Stop the node.  For servers that use ``init``, run the following command:
+
+   .. code-block:: console
+
+      # service mysql stop
+
+   For servers that use ``systemd``, instead run this command:
+
+   .. code-block:: console
+
+      # systemctl stop mysql
+
+#. Using your preferred text editor, ensure that the ``my.cnf`` configuration file includes parameters to secure the database server and client, (``mysqldump`` accesses the server through the client). 
+
+   .. code-block:: ini
+
+      # MySQL Server
+      [mysqld]
+      ssl-ca = /path/to/ca-cert.pem
+      ssl-key = /path/to/server-key.pem
+      ssl-cert = /path/to/server-cert.pem
+
+      # MySQL Client Configuration
+      [client]
+      ssl-ca = /path/to/ca-cert.pem
+      ssl-key = /path/to/client-key.pem
+      ssl-cert = /path/to/client-cert.pem
+
+#. Additionally, configure :ref:`wsrep_sst_auth <wsrep_sst_auth>` with the SST user authentication information.
+
+   .. code-block:: ini
+
+      [mysqld]
+      # mysqldump SST auth
+      wsrep_sst_auth = sst_user:sst_password
+
+This configures the node to use ``mysqldump`` for state snapshot transfers over SSL.  You must repeat this process on all other nodes in order for them to communicate securely with one another.  When all nodes are updated to SSL, you can begin restarting the cluster.  For more information on how to do this, see :doc:`startingcluster`.
```

Comment:
This configures mysqldump SST to use SSL. It does not make mysqldump the current SST method.